frontend/package.json

```diff
@@ -22,62 +22,64 @@
     "reinstall": "npm run purge && npm install",
     "serve": "vite",
     "test": "vitest run --coverage",
+    "test:dev": "vitest",
     "typecheck": "vue-tsc --noEmit"
   },
   "dependencies": {
     "@bcgov/bc-sans": "^2.1.0",
-    "@fortawesome/fontawesome-svg-core": "^6.5.1",
-    "@fortawesome/free-solid-svg-icons": "^6.5.1",
-    "@fortawesome/vue-fontawesome": "^3.0.6",
+    "@fortawesome/fontawesome-svg-core": "^6.6.0",
+    "@fortawesome/free-solid-svg-icons": "^6.6.0",
+    "@fortawesome/vue-fontawesome": "^3.0.8",
     "@geoman-io/leaflet-geoman-free": "^2.17.0",
     "@types/geojson": "^7946.0.14",
     "@types/leaflet": "^1.9.12",
     "@types/proj4": "^2.5.5",
-    "axios": "^1.6.8",
+    "axios": "^1.7.2",
     "date-fns": "^3.6.0",
-    "filesize": "^10.1.0",
+    "filesize": "^10.1.4",
     "geojson": "^0.5.0",
     "leaflet": "^1.9.4",
     "oidc-client-ts": "^3.0.1",
     "pinia": "^2.1.7",
     "pinia-plugin-persistedstate": "^3.2.1",
     "primeflex": "^3.3.1",
-    "primeicons": "^6.0.1",
+    "primeicons": "^7.0.0",
     "primevue": "3.52.0",
```

Comment:
Not sure if needed but may want to go to 3.53.0 for this PR as you've made a lot of dependency changes. I do have it as 3.53.0 in PR-117 so it will be merged.